### DISCUSSION TITLE
네이밍 관련 이야기를 나눕니다.

### DISCUSSION THREAD

**Discussion by jgjgill** [prgrms-fe-devcourse/FEDC4_HONKOK_JunilHwang#45] [General]
네이밍 관련 팀 규칙을 정하고 있지만 모호한 부분이 계속 존재하네요..
이와 관련해서 discussion이 있으면 좋겠다는 생각이 들어 새로 생성합니다.
자유롭게 의견 올려주세요..! 🙇‍♂️

**Comment by jgjgill** [comment]
현재 생성된 채널 목록을 가져오는 `useQuery`를 사용하려고 하는데요! 네이밍이 헷갈리네요..

`const { data: channelGetChannelsQuery?(질문 사항!) } = useQuery({ queryKey: [''], queryFn: channelService.getChannels });`

`channel`이 앞뒤로 붙는데 이러한 규칙으로 정했던 것 같아서 코멘트남깁니다.. 🤔

**Reply by hyoribogo** [reply]
이 경우에는 `channelsQuery`로 정하면 될 것 같아요 !!
